lib/node_modules/@stdlib/math/stats/anova1/package.json

```diff
@@ -0,0 +1,62 @@
+{
+  "name": "@stdlib/math/stats/anova1",
+  "version": "0.0.0",
+  "description": "Examine 1-way ANOVA relationships",
```

Comment:
Let's change the description to `Perform a one-way analysis of variance.`

Reply:
Resolved.